Packs/GoogleSafeBrowsing/Integrations/GoogleSafeBrowsingV2/GoogleSafeBrowsingV2.py

```diff
@@ -0,0 +1,242 @@
+import demistomock as demisto
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+
+import requests
+import traceback
+from typing import Dict, Any
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+TYPES = {
+    'threatTypes': ["MALWARE", "SOCIAL_ENGINEERING", "POTENTIALLY_HARMFUL_APPLICATION", "UNWANTED_SOFTWARE"],
+    'platformTypes': ["ANY_PLATFORM", "WINDOWS", "LINUX", "ALL_PLATFORMS", "OSX", "CHROME", "IOS", "ANDROID"]
+}
+
+INTEGRATION_NAME = 'GoogleSafeBrowsing'
+URL_OUTPUT_PREFIX = 'GoogleSafeBrowsing.URL'
+
+
+class Client(BaseClient):
+    def __init__(self, proxy: bool, verify: bool, reliability: str, base_url: str):
+        headers = {
+            'Content-Type': 'application/json',
+            'Accept': 'application/json'
+        }
+        super().__init__(proxy=proxy, verify=verify, base_url=base_url, headers=headers)
+
+        self.base_url = base_url
+        self.client_body = {
+            'clientId': demisto.params().get('client_id'),
+            'clientVersion': demisto.params().get('client_version'),
```

Comment:
please move the params to the class and do not call demisto.params() inside.

Reply:
done